modules/openy_gc_auth/modules/openy_gc_auth_reclique_sso/config/install/openy_gc_auth.provider.reclique_sso.yml

```diff
@@ -1,8 +1,11 @@
 authorization_server: 'https://[association_slug].recliquecore.com'
 client_id: ''
 client_secret: ''
-error_not_found: 'That user was not found.'
 error_access_denied: 'That user does not have access to Virtual Y.'
-error_invalid: 'Something went wrong.'
 error_accompanying_message: 'Please contact us if you have any questions.'
+error_invalid: 'Something went wrong.'
+error_not_found: 'That user was not found.'
 login_mode: 'present_login_button'
+membership_field: 'PackageName'
+permissions_mapping: ''
+require_active: 1
```

Comment:
We should have hook_update_N for this config to be updated

Reply:
@podarok these are all new fields and have default values https://github.com/YCloudYUSA/yusaopeny_gated_content/pull/4/files#diff-f494604cd8afa5f9c202ee02db5e7cfb894c493618f019fe30baf837c12ec481R102
so I believe from prior conversations they don't need an update hook.